src/components/Header/index.js

```diff
@@ -65,7 +64,7 @@ const styles = StyleSheet.create({
     fontFamily: fonts.poppins[600],
     fontSize: 16,
     fontWeight: '600',
-    marginRight: 15
+    marginRight: 50
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in `marginRight` from 15 to 50 is quite significant. If this change is intended for a specific screen size or device, consider making it responsive to avoid layout issues on different screen sizes.
```diff
-    marginRight: 50
+    marginRight: Dimensions.get('window').width > 500 ? 50 : 15
```
This will apply a `marginRight` of 50 if the screen width is greater than 500, and 15 otherwise. This is just an example, adjust the condition as per your requirements.


<!-- This is an auto-generated comment by OSS CodeRabbit -->